.gitignore

```diff
@@ -21,6 +21,7 @@ _local_/*
 .vscode/
 **/compile_commands.json
 .clangd/
+venv/
```

Comment:
```suggestion
```
I suggest instead to place your python venv inside the build directory

Reply:
I put that there intentionally, because my IDE requires it for python coding.
Is it ok if we leave it in?